eva-events-db-schema/src/main/resources/com/razz/eva/events/db/V001__create_events.sql

```diff
@@ -6,6 +6,7 @@ CREATE TABLE uow_events (
   idempotency_key       TEXT,
   principal_name        TEXT                NOT NULL,
   principal_id          TEXT                NOT NULL,
+  principal_context     TEXT                NOT NULL DEFAULT '{}',
```

Comment:
as we are not going to query events - I don't why we should use JSONB in this case

Reply:
why should it be not null?